Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.h

```diff
@@ -53,7 +53,15 @@ enum class PickerDismissalReason : uint8_t;
 - (NSArray<NSString *> *)currentAvailableActionTitles;
 - (NSArray<NSString *> *)acceptedTypeIdentifiers;
 
-+ (std::pair<RetainPtr<NSURL>, RetainPtr<NSURL>>)_copyToNewTemporaryDirectory:(NSURL *)originalURL fileCoordinator:(NSFileCoordinator *)fileCoordinator fileManager:(NSFileManager *)fileManager;
+enum class MovedSuccessfully : bool { No, Yes };
```

Comment:
Nit - let's put this new struct and enum class inside of the `WebKit` namespace, so that we won't collide with other symbols in the global namespace in the future.